src/components/molecules/ContentCard.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+	<BaseCard v-bind="$props">
+		<div class="content-card">
+			<template v:slot:content>
+				<div class="content">
+					<BaseLink :href="url" target="_blank">
+						<img
+							:src="thumbnail"
+							alt="content-thumbnail"
+							class="content__thumbnail"
+						/>
+						<div class="content__title">{{ title }}</div>
+					</BaseLink>
+					<div class="content__tags">
+						<span v-for="(tag, i) in tags" :key="i" class="content__tags-tag">
+							{{ tag }}
+						</span>
+					</div>
+					<div class="content__description">{{ description }}</div>
+				</div>
+			</template>
+			<template v:slot:footer>
+				<div class="footer">
+					<div class="footer__melden">
+						<a
+							href="mailto:inhalte@schul-cloud.org?Subject=Melden%20des%20Inhaltes%20mit%20der%20ID%20&body=Liebes%20%20Team%2C%0Ahiermit%20m%C3%B6chte%20ich%20den%20im%20Betreff%20genannten%20Inhalt%20melden%2C%20da%3A%0A%5Bhier%20bitte%20Ihre%20Gr%C3%BCnde%5D"
```

Comment:
could you please put this string into a computed property and generate the uri encoded version from a regular string? This would improve readability and changeability a lot.
Best case would be to use the language files for this string as well.

Something like this:
```suggestion
							:href="`mailto:inhalte@schul-cloud.org?${mailContent}`"
```

```vue
computed: {
  reportMail(){
    const mailContent = {
      Subject: this.$t("contentcard.report.subject"),
      body: this.$t("contentcard.report.body"),
    }
    return Object.keys(mailContent).map(key => key + '=' + encodeURIComponent(params[key])).join('&');
  }
}
```
add your language keys (`contentcard.report.subject`) to `locales/de.json` and use [our naming scheme for these keys](https://vuepress.test.schul-cloud.org/1-Tutorials/Localization.html#verwendung) (sorry, this part of the documentation is currenly only available in german)